crates/sui-framework/packages/sui-framework/sources/transfer.move

```diff
@@ -111,6 +114,18 @@ module sui::transfer {
         receive_impl(object::uid_to_address(parent), id, version)
     }
 
+    /// Given mutable (i.e., locked) access to the `parent` and a `Receiving` argument
+    /// referencing an object of type `T` owned by `parent` use the `to_receive`
+    /// argument to receive and return the referenced owned object of type `T`.
+    /// The object must have `store` to be received outside of its defining module.
+    public fun public_receive<T: key + store>(parent: &mut UID, to_receive: Receiving<T>): T {
```

Comment:
Random question, do we care about affecting the function indexes here? I suppose we don't but just checking :)

Reply:
No, I don't think so, because we clear the cache on every epoch, and the system packages are loaded afresh.